fundingmanager_test.go

```diff
@@ -657,25 +657,26 @@ func fundChannel(t *testing.T, alice, bob *testNode, localFundingAmt,
 
 	// Create a funding request and start the workflow.
 	errChan := make(chan error, 1)
-	initReq := &openChanReq{
```

Comment:
can delete the definition of `openChanReq`?

Reply:
Yes good idea